startup-script/startup-script.yml

```diff
@@ -0,0 +1,31 @@
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: startup-script
+  labels:
+    app: startup-script
+spec:
+  template:
+    metadata:
+      labels:
+        app: startup-script
+    spec:
+      hostPID: true
+      containers:
+        - name: startup-script
+          image: gcr.io/mikedanese-k8s/startup-script:v1
```

Comment:
From the above makefile it looks like the path here should be `gcr.io/google-containers/startup-script:v1`.